schedview/dayobs.py

```diff
@@ -31,15 +41,32 @@ class DayObs:
         ``yyyymmdd`` if an integer representation is a mapping of decimal
         digits to year, month, and day; ``mjd`` if the integer representation
         is the Modified Julian Date. The default is ``mjd``.
+    atmosphere : `dict` or `None`
+        A dictionary with atmospheric conditions. These are passed directly
+        as keyword arguments to the utilized `astropy.coordinates.AltAz` for
+        use in converting to alt and az.
+        Defaults to None, which is vaccuum.
+    location : `astropy.coordinates.EarthLocation`
+        The location of the observatory. Defaults to the Simonyi telescope.
     """
 
     # Use the standard library date as a lowest common denominator
     # that cannot represent more or less than we want it to.
     date: datetime.date
     int_format: IntDateFormat = "auto"
+    atmosphere: dict | None = None
+    location: EarthLocation = EarthLocation.from_geodetic(
+        lon=-70.7494 * u.deg, lat=-30.2444 * u.deg, height=2650.0 * u.meter
+    )
```

Comment:
I'd also be curious if this is different from the EarthLocation determined from Site("LSST").to_earth_location()

Reply:
This is almost a cut and paste from Site from rubin_scheduler, again because that's what I originally used by was trying to remove the dependency.